snapcraft/hooks/connect-plug-ceph-conf

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+ln -snf ${SNAP_DATA}/microceph/conf/ /etc/ceph
```

Comment:
I suppose we shouldn't do this if `ceph.external` is enabled?

Reply:
@masnax please can you fix this ASAP so I can merge and you can test a latest/edge build for your testing.

Reply:
Fixed now, I had to copy over a helper from `configure` into each hook because I couldn't figure out how to share function and variable definitions between hooks.

Reply:
Oh, I just noticed I didn't actually add the commit :)

Reply:
Fixed now. @tomponline this should be ready now, thanks.